src/main/java/de/filefighter/rest/domain/user/business/UserBusinessService.java

```diff
@@ -148,6 +138,15 @@ public boolean passwordIsValid(String password) {
         return pattern.matcher(password).matches();
     }
 
+    /**
```

Comment:
haha
should be host the javadocs on our server?

Reply:
we could host it on the blog maybe. But i mean we have enough todo right xD